### DISCUSSION TITLE
Best practices for adding imports

### DISCUSSION THREAD

**Discussion by chisaato** [nuxt/nuxt#16617] [Questions]
![image](https://user-images.githubusercontent.com/17796830/195922768-8268d428-3b7f-4538-ba62-a4354da781d8.png)

as it says there are 3 way to import `useRoute()` which is the best practice?

I need to use `useRoute()` API from [here](https://v3.nuxtjs.org/api/composables/use-route)

**Comment by TheAlexLichter** [comment]
If you use Nuxt's [auto import](https://v3.nuxtjs.org/guide/concepts/auto-imports) feature, which is enabled by default, you don't need any import at all ☺️
